packages/rs-drive-abci/src/execution/platform_events/core_based_updates/update_quorum_info/v0/mod.rs

```diff
@@ -154,6 +157,117 @@ where
                 }
             });
 
+        if validator_set_quorum_type == chain_lock_quorum_type {
```

Comment:
How is it possible that both chain locks and validators on the same quorum type?

Reply:
I mean... it is for testnet right?

Reply:
Yes, technically you can configure them to use the same type so yeah we need to support it